cdap-hbase-compat-1.1/src/main/java/co/cask/cdap/data2/replication/hbase11/LastWriteTimeObserver.java

```diff
@@ -71,8 +69,6 @@ public void postWALWrite(ObserverContext<? extends WALCoprocessorEnvironment> ct
               logKey.getTablename().toString(),
               logKey.getWriteTime(),
               logKey.getEncodedRegionName().toString());
-    if (hBase11TableUpdater != null) {
-      hBase11TableUpdater.updateTime(new String(logKey.getEncodedRegionName(), "UTF-8"), logKey.getWriteTime());
-    }
+    hBase11TableUpdater.updateTime(new String(logKey.getEncodedRegionName(), "UTF-8"), logKey.getWriteTime());
```

Comment:
did you see the earlier comments about this? In general, you can't just assume a byte[] is a utf8 string. From a brief look at the code, it looks like you should be using HRegionInfo.encodeRegionName().